scheduler/test/cook/test/mesos/pool.clj

```diff
@@ -0,0 +1,22 @@
+(ns cook.test.mesos.pool
+  (:require [clojure.test :refer :all]
+            [cook.config :as config]
+            [cook.mesos.pool :as pool])
+  (:import (clojure.lang ExceptionInfo)))
+
+(deftest test-guard-invalid-default-pool
```

Comment:
Can you regex match on the message strings of the exceptions to verify the expected path was triggered?

Reply:
Done